server/src/app/api/create/create_activity.tsx

```diff
@@ -0,0 +1,29 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
This should be a `.ts` file if you're not including any tsx markup.

Reply:
Noted